phpBB/install/phpbbcli.php

```diff
@@ -39,6 +39,9 @@
 // Enable superglobals for cli support
 $phpbb_installer_container->get('request')->enable_super_globals();
 
+// Load compatibility globals to access deprecated globals
+require($phpbb_root_path . 'includes/compatibility_globals.' . $phpEx);
```

Comment:
Is there a reason not to call that or is it that you just don't see a point in it right now? If it's the latter I'd recommend just calling it ;)